src/omniscapeTransformer.py

```
@@ -850,6 +903,28 @@ def cleanup_julia_process():
+)
+if write_modified:
+    last_mod_idx = len(resistanceModifiers) - 1
+    if is_tiling:
+        tile_mod_paths = [
+            os.path.join(dataPath, "omniscape_ResistanceModifier", f"tile-{tid}-mod{last_mod_idx}-resistance.tif")
+            for tid in tiles_to_process
+        ]
+        merged_mod_path = os.path.join(dataPath, "omniscape_ResistanceModifier", "resistance-modified.tif")
+        merge_tile_outputs(tile_mod_paths, merged_mod_path)
+        myOutput.modifiedResistance = pd.Series(merged_mod_path)
+        ps.environment.update_run_log(f"  Added modifiedResistance output (merged tiles)")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)